misc/utility/color.py

```diff
@@ -111,20 +111,20 @@ def print_info(*values: object) -> None:
     if _stdout_override:
         print(f"{Ansi.GRAY}{Ansi.BOLD}INFO:{Ansi.REGULAR}", *values, Ansi.RESET)
     else:
-        print(*values)
+        print("INFO:", *values)
```

Comment:
```suggestion
        print("INFO: ", *values)
```

Reply:
`print()` already puts spaces between its arguments, see the colored version above.

```
>>> print("INFO:", "Super important")
INFO: Super important
```